docs/runtimes/databricks.ipynb

```diff
@@ -0,0 +1,117 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(databricks)=\n",
+    "# Databricks runtime\n",
+    "\n",
+    "The databricks runtime runs on a Databricks cluster (and not in the Iguazio cluster). With the databricks runtime, you \n",
+    "send keyword arguments (kwargs) to this job, send your local file/code as a string to the job, and use a handler \n",
+    "as an endpoint for user code.\n",
+    "\n",
+    "You can run the function on:\n",
+    "- An existing cluster, by including the `DATABRICKS_CLUSTER_ID`\n",
+    "- A job compute cluster, created and dedicated for this function only. Omit `DATABRICKS_CLUSTER_ID` to create a job compute cluster, and set the [cluster specs](https://docs.databricks.com/en/workflows/jobs/jobs-2.0-api.html#newcluster) by using the task parameters, when running the function, For example:\n",
+    "   ```\n",
+    "   params['task_parameters'] = {'new_cluster_spec': {'node_type_id': 'm5d.xlarge', 'num_workers': 2, 'timeout_minutes': 15}}\n",
+    "   ```\n",
+    "Do not send variables named `mlrun_internal_code` or `context` since these are utilized by the internal processes of the runtime.\n",
```

Comment:
we no longer use `mlrun_internal_code`, instead of it we pass all task parameters as a part of the dict:task_parameters.

so please replace:

```suggestion
    "Do not send variables named `task_parameters` or `context` since these are utilized by the internal processes of the runtime.\n",
```